tartiflette/parser/nodes/field.py

```diff
@@ -46,7 +47,7 @@ def _to_dict(thing):
 
 class NodeField(Node):
     def __init__(self, func, location, path, name, type_condition):
-        super(NodeField, self).__init__(path, 'Field', location, name)
+        super().__init__(path, 'Field', location, name)
```

Comment:
IMHO it's more consistent to keep the super fully explicit, cause if we double inherit sometimes, it won't creates confusion. Explicit > *.

Reply:
As seen with bubu, it's OK. https://stackoverflow.com/questions/26927571/multiple-inheritance-in-python3-with-different-signatures :D